sys/compat/freebsd32/freebsd32_misc.c

```diff
@@ -1002,14 +1002,15 @@ freebsd32_ptrace(struct thread *td, struct freebsd32_ptrace_args *uap)
 }
 
 static int
-freebsd32_copyinuio(struct iovec32 * __capability iovp, u_int iovcnt,
+freebsd32_copyinuio(const void * __capability _iovp, u_int iovcnt,
```

Comment:
Hm, never been a huge fan of this style...

Reply:
Happy to change it, just seemed like the easiest way to get signatures checked.

Reply:
What style would you suggest? As there is only one use of the variable I could also add a cast there?

Reply:
I would leave the existing typedefs for the callbacks and instead use local variables with the 32/64 pointer types that use a cast to assign them at the start of the functions.  I know it is a bigger diff, but I would probably call those local pointers `iovp32` and `iovp64`, etc. instead of making the cast source `_iovp`.

So for example, here you would have:

```
static int
freebsd32_copyinuio(const struct iovec * __capability iovp, u_int iovcnt,
    struct uio **uiop)
{
    const struct iovec32 * __capability iovp32 = (const struct iovec32 * __capability)iovp;
...
```